bundles/org.openhab.core.automation.module.script/src/main/java/org/openhab/core/automation/module/script/internal/provider/ScriptModuleTypeProvider.java

```diff
@@ -53,7 +55,7 @@
 public class ScriptModuleTypeProvider implements ModuleTypeProvider {
 
     private final Logger logger = LoggerFactory.getLogger(ScriptModuleTypeProvider.class);
-    private final List<ParameterOption> parameterOptions = new CopyOnWriteArrayList<>();
+    private final Map<String, String> parameterOptions = new TreeMap<>();
```

Comment:
One more question: The `CopyOnWriteArrayList` is thread-safe. `TreeMap`is not. Do you think that might be a problem?

Reply:
I mentioned this in #1253. I don't see it as an issue, but if you can suggest a safer way I'll put it in. I like the TreeMap, so that the entries are unique and sorted. I've never had more than one ScriptEngineFactory start at the same time, and I not see how more than one could be possible.

Reply:
I am fine with a `TreeMap`.

Reply:
If errors are reported, I'll figure out how to build the map safely.